pkg/pillar/types/patchenvelopestypes_test.go

```diff
@@ -0,0 +1,113 @@
+package types
```

Comment:
Also missing Copyright and License lines.

Reply:
Now you can see it :)

Reply:
yes.